app/views/layouts/_header.html.slim

```diff
@@ -1,14 +1,57 @@
-header.navbar.navbar-fixed-top.navbar-dark
-  div.container
-    = link_to 'SalesAssistant', new_session_path(resource_name), id: 'logo'
-    nav
-      ul.nav.navbar-nav.navbar-right
-        - if user_signed_in?
-          li.nav_li
-            = link_to 'Logout', destroy_user_session_path, method: :delete
-        - else
-          - if controller_name != 'sessions'
-            li.nav_li
-              = link_to  new_session_path(resource_name) do
-                i.fa.fa-user
-                  | Login
\ No newline at end of file
+nav.header-navbar.navbar-expand-md.navbar.navbar-with-menu.navbar-static-top.navbar-dark.navbar-border
+  .navbar-wrapper
+    .navbar-container.content
+      = link_to 'SalesAssistant', new_session_path(resource_name), id: 'logo'
+      - if user_signed_in?
+        #navbar-mobile.collapse.navbar-collapse.float-right
+          ul.nav.navbar-nav.float-right
+            li.dropdown.dropdown-notification.nav-item
+              a.nav-link.nav-link-label[href="#" data-toggle="dropdown"]
+                i.ficon.fa.fa-bell
+                span.badge.badge-pill.badge-default.badge-danger.badge-default.badge-up
+                  | 0
+              ul.dropdown-menu.dropdown-menu-media.dropdown-menu-right
+                li.dropdown-menu-header
+                  h6.dropdown-header.m-0
+                    span.grey.darken-2
+                      | Messages
+                    span.notification-tag.badge.badge-default.badge-warning.float-right.m-0
+                      | Empty
+            li.dropdown.dropdown-notification.nav-item
+              a.nav-link.nav-link-label[href="#" data-toggle="dropdown"]
+                i.ficon.fa.fa-envelope
+                span.badge.badge-pill.badge-default.badge-warning.badge-default.badge-up
+                  | 0
+              ul.dropdown-menu.dropdown-menu-media.dropdown-menu-right
+                li.dropdown-menu-header
+                  h6.dropdown-header.m-0
+                    span.grey.darken-2
+                      | Messages
+                    span.notification-tag.badge.badge-default.badge-warning.float-right.m-0
+                      | Empty
+            li.dropdown.dropdown-user.nav-item
+              a.dropdown-toggle.nav-link.dropdown-user-link[href="#" data-toggle="dropdown"]
+                span.avatar.avatar-online
+                  = gravatar_image_tag(current_user.email)
+                span.user-name
+                  = current_user.first_name
```

Comment:
maybe easier to create a method in User 
full_name and just output here span.user-name= current_user.full_name? 
rework